src/layers/createMarkerSVG.ts

```diff
@@ -14,9 +14,8 @@ export function createSvg({ color, number, size = 0 }: MarkerProps) {
     return `<svg aria-hidden="true" focusable="false" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 384 512" width="${
         // todo: we do not use width in Marker.tsx, but without this the markers are not shown in Firefox :( (but they are shown in Chrome...)
         size
-    }" height="${size}" style="cursor: pointer; stroke: none; fill: ${hexToRgb(color)}; transform: \`translate(${
-        -size / 2
-    }px, ${-size}px)"><path d="${MARKER_PATH}"/>${
```

Comment:
This `transform: translate` function was also broken (see my comment in `Marker.tsx`). It had the closing parenthesis, but there was an extra backtick before `translate`. It seems like we can use `translate` to move the markers, but then we won't see the entire marker any longer because it is moved out of the svg's `viewBox`. I therefore used the `displacement` option for the `Icon` in `UseQueryPointsLayer` instead.